samples/net/sockets/http_server/sample.yaml

```diff
@@ -13,6 +13,5 @@ common:
   platform_exclude:
     - native_posix
     - native_posix/native/64
-    - nrf5340dk/nrf5340/cpuapp/ns # Excluding due to ROM overflow.
```

Comment:
Why was this removed, the sample will still not fit to that platform?

Reply:
Yes, correct. I initially thought that moving from legacy Mbed TLS support to TF-M's PSA Crypto on that platform could make the sample build successfully, but I just tried and it's not the case. So I need to revert this leftover. Thanks for noticing.